Sample/O365-iOS-Snippets/MasterViewController.m

```diff
@@ -481,6 +486,261 @@ - (void)performDeleteCalendarEvent
                                 }];
 }
 
+- (void) performAcceptCalendarEvent{
```

Comment:
General comment: we should start summarizing what these methods do. There is setup that these methods perform that may be of interest to reader. Inside each of these methods, we should comment where a method call results in a request to the service.